include/seqan3/range/detail/inherited_iterator_base.hpp

```diff
@@ -159,7 +183,7 @@ class inherited_iterator_base : public base_t
     }
 
     //!\brief Post-decrement, return previous iterator state.
-    constexpr derived_t operator--(int) noexcept(noexcept(--base_t{}))
+    constexpr derived_t operator--(int) noexcept(noexcept(--std::declval<base_t &>()))
```

Comment:
```suggestion
    constexpr derived_t operator--(int) noexcept(noexcept(std::declval<base_t &>()--))
```

Reply:
no, since it is implemented via pre-increment, the noexcept status depends also on pre-increment. However of the derived type.

Reply:
Right, but then the copyconstruction must be explicilty tested, which you did now  right?

Reply:
yes, although it's not copy construction, it's construction-from-base_t

Reply:
Even better to test for it explicitly.